Karl/karl.h

```diff
@@ -313,17 +463,17 @@ class GenericKarl final {
             }
           }();
 
+          const auto now = current::time::Now();
```

Comment:
Perhaps call it some `now_for_time_skew_computation` then? A bit confusing as is, with two `now` variables declared one in the outer and one in the inner scope.